docs/UserGuide.md

```diff
@@ -1,33 +1,94 @@
-# User Guide
+# ra.VI User Guide
 
 ## Introduction
 
-{Give a product intro}
+Repository Assistant with a Versatile Interface (ra.VI) is a desktop app for managing tasks, deadlines, and notes, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ra.VI can manage your tasks faster than traditional GUI apps.
 
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of `ra.VI` from [here](http://link.to/duke).
+3. Copy the file to the folder you want to use as the home folder for your scheduler. Double-click the file to start the app. 
+4. Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.  
+Some example commands you can try:
+* `add -t`: Add a task
+* `list -t`: List all tasks
+* `exit`: Exit the program read chapter 1`
+5. Refer to the Features below for details of each command
 
 ## Features 
 
 {Give detailed description of each feature}
+### Viewing help: `help`
+Shows a message with the list of available commands and functions.
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+### Adding an item: `add <opt><args>`
+#### Adding a task: `add -t [-by]`
+Add a task to the scheduler.  
+Format: `add -t <task_name> [-by <deadline>]`
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+Example of usage: 
+* `add -t read chapter 1`
+* `add -t read chapter 1 -by 12 Sep`
```

Comment:
Can we actually use this format for date?

Reply:
Format for the deadline is d-M-yyyy HHmm\
e.g. add -t read chapter 1 -by 12-9-2020 1400